### DISCUSSION TITLE
Indexing / Folder / Image name / Missing Features

### DISCUSSION THREAD

**Discussion by thomaslc66** [photoprism/photoprism#3871] [General]
Hello everybody,

I'm new to this tool and I can't figure out some of the basic stuff and UI implementation. I think maybe it's just that I'm new and you might help me here but I've struggled to find basic stuff in the UI.

So I've managed to Sync Photos folder from Nextcloud using webdav to PhotoPrism, nothing special I've selected Download remote file and unchecked the "Keep filename" as I though it was a mess in my current Photos folder and this tool would help me reorganize.

But now I'm here, facing a folder with today's date and all the picture that this tool didn't manage to find the date or anything else and I'm wondering:
1) How can I move thoses files in their respective folders? The folder is called November 2023 and the pictures need to be in October 2005
2) As the picture have been renamed with a timestamp and as this timestamp is wrong, will they be renamed if I manage to move them to the correct folder and change their details date? Fom 2023/11/20231107_164721_3240BA13.jpg to 2005/05/20050505_XXXX_XXXX.jpg ???

Also, I wanted to know if it's only me or if this tool is missing some basic features like a "Select all or Select multiple" pictures option, or a one-click delete/archive button and so one.

I'll thank you in advance for all your feedback and help.

Regards,
Thomas

**Comment by lastzero** [comment]
https://docs.photoprism.app/user-guide/navigate/#selection-mode-and-multi-select

https://docs.photoprism.app/getting-started/faq/#can-i-use-photoprism-to-sort-files-into-a-configurable-folder-structure

So while the import functionality automatically generates a unique filename based on the metadata (that exists at import time), there is currently no feature that would move files to different folders from the UI. Of course, you can manually delete, rename, and modify pictures in the file system (or use any tool for this that does that automatically) and then reindex your library to apply the changes.

**Reply by thomaslc66** [reply]
Thanks for the update.

What will happen if I rename them put them in the correct folder without changing the metadata ? Will they be put back in the same folder with the old name ?

Also, is there not something already implemented that will do that based on the edited metadata/details in the UI.
I though that if I edit the details add a date and a place and then reindex it will change the folder and file name automatically.

**Reply by lastzero** [reply]
1. No, they are not automatically moved back.
2. This could be implemented, for example, with a relatively simple CLI command as a first step. Since our core team is currently working on multi-user functionality, we would be happy about contributions.
